draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +200,26 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## CORE Beaconing Scalability
+
+The idea behind beaconing is to discover all possible paths (loop free and with a fixed maximu length) between two CORE AS.
+Every AS forwards any received becons to all neighbor CORE ASes unless this would cause a loop or exceed the fixed maximum length.
+Implemented naively, the number of paths (and beacons) grows exponentially with the network size.
+This is currently mitigated primarily (and efficiently) by forwarding only the 5 "best PCB" to neighbours.
+
+This "best PCB" practive has several implications:
+
+* It limits the number of available paths for endhosts such that only 5 path may be available to a given destination. This is especially relevant for
```

Comment:
```suggestion
* It limits the number of available paths for endpoints such that only 5 path may be available to a given destination. This is especially relevant for
```

In drafts, we use `endpoint`  as per definition in RFC9473. Could you generally use endpoint instead of end host?